charts/stardog/values.yaml

```diff
@@ -53,6 +53,15 @@ resources: {}
 #    cpu: 2
 #    memory: 6Gi
 
+securityContext:
+  enabled: false
+  runAsUser: null
+  runAsGroup: null
+  fsGroup: null
+
+initContainers:
+  enabled: true
```

Comment:
can you add some comments to all of these briefly describing how/why they're used or why they'd be changed?

Reply:
good idea, shall do.

Reply:
done